### PR TITLE
Mobile Navigation - Video Streaming 4.7 Updates

### PR DESCRIPTION
**Pull request by bilal-alsharifi**

### CODE REVIEW

File: docs/Mobile Navigation/Video Streaming/index.md [hunk 1]
```diff
@@ -39,21 +36,33 @@ public static class MyDisplay extends SdlRemoteDisplay{
 }
 ```
 
+!!! Note
+If you are obfuscating the code in your app, make sure to exclude you `SdlRemoteDisplay` class from being obfuscated. For more information on how to do that, you can check [Proguard Guidelines](/guides/android/proguard-guidelines/).
```
Comment (BrettyWhite) [inline docs/Mobile Navigation/Video Streaming/index.md]:
```suggestion
If you are obfuscating the code in your app, make sure to exclude your `SdlRemoteDisplay` class. For more information on how to do that, you can check [Proguard Guidelines](/guides/android/proguard-guidelines/).
```

File: docs/Mobile Navigation/Video Streaming/index.md [hunk 2]
```diff
@@ -39,21 +36,33 @@ public static class MyDisplay extends SdlRemoteDisplay{
...
+!!!
+
 ## Managing the Stream
```
Comment (BrettyWhite) [inline docs/Mobile Navigation/Video Streaming/index.md]: maybe we should give an example of a `MyDisplay` class. We reference it a lot.

File: docs/Migrating to Newer SDL Android Versions/Updating to 4.7/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+
+
+
+
+## Video Streaming:
+
+Previously, developers had to make sure that the app was in HMI_FULL before starting the video stream, In 4.7, the developer can start video streaming in `VideoStreamingManager.start()`'s `CompletionListener`. The `VideoStreamingManager` will take care of starting the video when the app becomes ready.
```
Comment (joeygrover) [inline docs/Migrating to Newer SDL Android Versions/Updating to 4.7/index.md]:
```suggestion
Previously, developers had to make sure that the app was in HMI_FULL before starting the video stream, In 4.7, after the `SdlManager` has called its `onStart` method, the developer can start video streaming in `VideoStreamingManager.start()`'s `CompletionListener`. The `VideoStreamingManager` will take care of starting the video when the app becomes ready.
```

File: docs/Mobile Navigation/Video Streaming/index.md [hunk 4]
```diff
@@ -39,21 +36,33 @@ public static class MyDisplay extends SdlRemoteDisplay{
 }
 ```
 
+!!! Note
+If you are obfuscating the code in your app, make sure to exclude your `SdlRemoteDisplay` class. For more information on how to do that, you can check [Proguard Guidelines](/guides/android/proguard-guidelines/).
```
Comment (joeygrover) [inline docs/Mobile Navigation/Video Streaming/index.md]:
```suggestion
If you are obfuscating the code in your app, make sure to exclude your class that extends `SdlRemoteDisplay`. For more information on how to do that, you can check [Proguard Guidelines](/guides/android/proguard-guidelines/).
```

File: docs/Mobile Navigation/Video Streaming/index.md [hunk 5]
```diff
@@ -39,21 +36,33 @@ public static class MyDisplay extends SdlRemoteDisplay{
...
+!!!
+
 ## Managing the Stream
-The only thing left to do to start the stream is combine the `SDLProxyALM` instance and the extension of the `SdlRemoteDisplay`. This should happen when your app receives its first `HMI_FULL` status in the `onOnHMIStatus(OnHMIStatus notification) ` callback. The method that needs to be called is `startRemoteDisplayStream(Context context, final Class<? extends SdlRemoteDisplay> remoteDisplay, final VideoStreamingParameters parameters, final boolean encrypted)` from the `SDLProxyALM`.
+To start streaming video, you can use `VideoStreamingManager`. The method that needs to be called is `startRemoteDisplayStream(Context context, final Class<? extends SdlRemoteDisplay> remoteDisplay, final VideoStreamingParameters parameters, final boolean encrypted)`.
```
Comment (joeygrover) [inline docs/Mobile Navigation/Video Streaming/index.md]:
```suggestion
The `VideoStreamingManager` can be used to start streaming video after the `SdlManager` has successfully been started. This is performed by calling the method `startRemoteDisplayStream(Context context, final Class<? extends SdlRemoteDisplay> remoteDisplay, final VideoStreamingParameters parameters, final boolean encrypted)`.
```

File: docs/Mobile Navigation/Video Streaming/index.md [hunk 6]
```diff
@@ -39,21 +36,33 @@ public static class MyDisplay extends SdlRemoteDisplay{
...
-    }
-   
+    });
 }
-
 ```
 
 ### Ending the Stream
-When the `HMIStatus` is back to `HMI_NONE` it is time to stop the stream. This is accomplished through a method `stopRemoteDisplayStream()` in the `SDLProxyALM`.
+When the `HMIStatus` is back to `HMI_NONE` it is time to stop the stream. This is accomplished through a method `stopStreaming()`.
```
Comment (joeygrover) [inline docs/Mobile Navigation/Video Streaming/index.md]: Does the developer need to listen for HMI events in this case? Or does the VideoStreamingManager take care of the stream based on HMI levels?

Comment (bilal-alsharifi) [reply docs/Mobile Navigation/Video Streaming/index.md]: The current implementation of `VideoStreamingManager` doesn’t stop streaming automatically when the app is in HMI_NONE. So the developer has to do that.

**Pull request merged**